server/analytics.js

```diff
@@ -1,7 +1,91 @@
 const { OperationsLayer } = require('./operations')
+const Neighborhood = require('./neighborhood')
 
 function filterByAll(constraintArray) {
 	return OperationsLayer.filterByAll(constraintArray);
 }
+function getColumn(constraintArray,x){
+	let list=OperationsLayer.filterByAll(constraintArray);
+	let column=[]
+	switch(x){
+		case 1:
+			for(let i=0;i<list.length;i++){
+				column.push(parseInt(list[i].median_value));

```

Comment:
Let's make this better and use a `.forEach()` loop. Ditto with the ones down below.

See the documentation here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
Switched to forEach